app/participants/admit.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations

```

Comment:
`admit.py` is not a self-describing filename, you would not understand what content is placed here by its name. Rename this file to `support_functions.py`, when this application expands you could place other support functions in this file.